appraisal/ruby-3.0.rb

```diff
@@ -115,6 +115,7 @@
   gem 'concurrent-ruby'
   gem 'dalli', '>= 3.0.0'
   gem 'grpc', '>= 1.38.0', platform: :ruby # Minimum version with Ruby 3.0 support
+  gem 'karafka'
```

Comment:
karafka no longer supports 3.0 based on: https://karafka.io/docs/Versions-Lifecycle-and-EOL/